Gruntfile.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function(grunt) {
```

Comment:
Since there is only one task (linting) I have set grunts default task to it. `./node_modules/.bin/grunt jshint` will also work but is more typing.